### PR TITLE
Adds a copy of the schema in the fs and links to it from the README

### LINKED ISSUE #141: Where's the schema.graphql file?
This being the premier GraphQL example, I expected the schema to be defined (and documented) in a GraphQL schema file.  Or maybe I overlooked something?  If not, it would be nice to have to support languages other than JS.  Thanks!

### PR DESCRIPTION
**Pull request by orta**
Fixes #141 - which I think is pretty legit, it's nice to be able to see the schema that's generated.

### CODE REVIEW

**Comment by acao** [comment]
I remember referencing the source for this project before SDL existed, 😆 .
agreed though

File: scripts/store-schema.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { printSchema } from 'graphql/utilities';
+import { writeFileSync } from 'fs'
+
+try {
+  var output = printSchema(swapiSchema);
+  writeFileSync('schema.graphql', output, 'utf8')
+} catch (error) {
+  console.error(error);
+  console.error(error.stack);
+}
```
Comment (benjie) [inline scripts/store-schema.js]: What does this try/catch solve? Does it give you more information about the errors?

Comment (orta) [reply scripts/store-schema.js]: I just kept it fromthe other script which acts the same, unsure

Comment (benjie) [reply scripts/store-schema.js]: Fair enough; I’m guessing it’s required to see the error metadata like location etc. 🤷‍♂️

**Pull request merged**